puppet/zulip/manifests/postgres_common.pp

```diff
@@ -46,6 +46,18 @@
       exec {'pip3_deps':
         command => 'python3 -m pip install pytz python-dateutil'
       }
+      file { '/usr/bin/pg_config':
+        ensure => 'link',
+        target => "/usr/pgsql-${zulip::base::postgres_version}/bin/pg_config"
+      }
```

Comment:
Where do we use `pg_config`?

Reply:
`pg_config` is used during the installation of `psycopg2`.

Reply:
And so, this symlink-in-Puppet solution is not optimal because `psycopg2` setup happens before `puppet-apply` and will err because `pg_config` is not present. The symlink creation must happen before `setup_venv` step.

Reply:
Resolved because I moved the symlink of `pg_config` to `create-production-venv` before a `puppy-apply`